### ISSUE TITLE
`$global` broken when using `template.stream()`

### ISSUE THREAD

**Issue by GenaANTG** [marko-js/marko#381]
Hi there!
This code doesn't work now. Before he worked fine.
```
let HomePage = require('./home.marko');
let response = {
        $global: {
            active: 'home',
            logged: logged,
            moment: moment,
        },
        currentPage: currentPage,
};
HomePage.stream(response).pipe(res);
```
I have a layout and his part(home.marko). I load a part of layout and give him the data to rendering, but nothing.

In a layout i tried to print data with: `${out.global.active}`, what i doing wrong? Previously everything worked fine.

**Comment by patrick-steele-idem** [comment]
Looks like a bug. I am investigating and should have an update soon. Thanks for opening an issue!

**Comment by patrick-steele-idem** [comment]
Should be fixed now. We were missing a test case that should have caught an internal API change. New version published with fix: `marko@3.11.7`

Thanks, again.

**Comment by GenaANTG** [comment]
@patrick-steele-idem Thanks a lot!
